src/htm/algorithms/SDRClassifier.hpp

```diff
@@ -21,6 +21,13 @@
 
 /** @file
  * Definitions for the SDR Classifier & Predictor.
+ * 
+ * `Classifier` learns mapping from SDR->input value (encoder's output). This is used when you need to "explain" the HTM network back to real-world, ie. mapping SDRs 
```

Comment:
Nitpick: It's hard to read lines which are this long...

Reply:
Thanks.  As a general rule of thumb, if I expect people to read something then it should be no more than 80 characters per line.  I think it improves readability.  This is important for documentation, but less so for source code.